Makefile

```diff
@@ -125,15 +129,65 @@ test: examples test-stable
 analyse-stack-sizes:
 	cargo stack-sizes $(release) --example $(EXAMPLE) $(features) -- -Z emit-stack-sizes
 
+
+
 .PHONY: apollo3
 apollo3:
 	LIBTOCK_PLATFORM=apollo3 cargo run --example $(EXAMPLE) $(features) \
-		--target=thumbv7em-none-eabi $(release)
+		--target=thumbv7em-none-eabi $(release) --
```

Comment:
The `--` argument is unnecessary here, please remove it.

Reply:
> I think that adding new boards should be done in a separate PR.

I think you are right, I'll move them in a separate PR